Documentation/German/Beispiel_Output.txt

```diff
@@ -0,0 +1,264 @@
+Importing Python Packages - please wait.

```

Comment:
hi @LeonBew - why do you also include this file here? Just want to make sure you want it part of this PR.

Reply:
Ah never mind, it is the sample data. Thanks! :)